File: app/src/main/java/com/sagar/watchnext/network/models/tv/reviews/Reviews.java

package com.sagar.watchnext.network.models.tv.reviews;

import java.util.List;

import com.google.gson.annotations.SerializedName;

public class Reviews {

    @SerializedName("id")
        private int id;

    @SerializedName("page")
        private int page;

    @SerializedName("results")
        private List<Review> reviews ;

    @SerializedName("total_pages")
        private int totalPages;

    @SerializedName("total_results")
        private int totalResults;

}

File: app/src/main/java/com/sagar/watchnext/network/models/tv/reviews/Reviews.kt
package com.sagar.watchnext.network.models.tv.reviews

import com.google.gson.annotations.SerializedName

class Reviews {

    @SerializedName("id")
    val id: Int = 0

    @SerializedName("page")
    val page: Int = 0

    @SerializedName("results")
    val reviews: List<Review>? = null

    @SerializedName("total_pages")
    val totalPages: Int = 0

    @SerializedName("total_results")
    val totalResults: Int = 0
}
